debezium-connector-postgres/src/main/java/io/debezium/connector/postgresql/PostgresType.java

```diff
@@ -0,0 +1,112 @@
+/*
+ * Copyright Debezium Authors.
+ *
+ * Licensed under the Apache Software License version 2.0, available at http://www.apache.org/licenses/LICENSE-2.0
+ */
+package io.debezium.connector.postgresql;
+
+/**
+ * A class that binds together a PostgresSQL OID, JDBC type id and the string name of the type.
+ * The array types contain link to their element type.
+ *
+ * @author Jiri Pechanec
+ *
+ */
+public class PostgresType {
+
+    public static final PostgresType UNKNOWN = new PostgresType("unknown", -1, Integer.MIN_VALUE); 
+
+    private final String name;
+    private final int oid;
+    private final int jdbc;
+    private final PostgresType element;
+
+    public PostgresType(String name, int oid, int jdbc, PostgresType element) {
+        super();
+        this.name = name;
+        this.oid = oid;
+        this.jdbc = jdbc;
+        this.element = element;
+    }
+
+    public PostgresType(String name, int oid, int jdbc) {
+        this(name, oid, jdbc, null);
+    }
+
+    /**
+     * @return true if this type is an array
+     */
+    public boolean isArray() {
+        return element != null;
+    }
+
+    /**
+     *
+     * @return symbolic name of the type
+     */
+    public String getName() {
+        return name;
+    }
+
+    /**
+     *
+     * @return PostgreSQL OID of this type
+     */
+    public int getOid() {
+        return oid;
+    }
+
+    /**
+     *
+     * @return JDBC id of the type as reported by JDBC metadata
+     */
+    public int getJdbc() {
+        return jdbc;
+    }
+
+    /**
+     *
+     * @return the type of element in arrays or null for primitive types 
+     */
+    public PostgresType getElement() {
```

Comment:
Can we name it `getElementType()` or `getComponentType()`? I used "component type" originally as that's the term known in the Java world. But both works for me.